app/views/idv/in_person/ssn.html.erb

```diff
@@ -1 +1 @@
-<%= render 'idv/shared/ssn', flow_session: flow_session, success_alert_enabled: false, updating_ssn: updating_ssn %>
+<%= render 'idv/shared/ssn', flow_session: flow_session, success_alert_enabled: false, updating_ssn: updating_ssn, threatmetrix_session_id: threatmetrix_session_id %>
```

Comment:
Where is this local assign used? It looks like the shared view references the flow session value directly.

https://github.com/18F/identity-idp/blob/75c5dbb40863ccb4ed24be6eabcd2c181f766939/app/views/idv/shared/_ssn.html.erb#L33

Maybe we don't need this change? (And should it also be removed from the `doc_auth` version of this view?)

Reply:
Yeah, the shared view is not currently referencing this--but I think it should be. We currently depend on `threatmetrix_session_id` being generated in `extra_view_variables`, so it would make sense to have the shared view refer to that, since it'll be there anyway.

Reply:
That sounds good to me. Thanks for making those changes to the view.